README.md

```diff
@@ -1,22 +1,27 @@
 Official Tyk Pump Docker Build
 ==============================
 
-This container only contains the Tyk Pump, an application that facilitates moving data between Tyk and Tyk Dashboard
+This container only contains the Tyk Pump, an application that facilitates moving data between the Tyk Gateway and the Tyk Dashboard
 
-Tyk Pump will run with a default configuration unless it has been overridden with the -v flag. A sample configurations have been provided to run Tyk Pump with the assumed linked containers under `mongo` and `redis`
+Tyk Pump will run with a default configuration unless it has been overridden with the -v flag. A sample configurations have been provided to run Tyk Pump with the assumed linked containers under `mongo` and `redis`.
+
+The following ports are required to be open:
+
+For Redis: 6379
+For MongoDB: 27017
 
 Quickstart
 ----------
 
-1. Get a redis container (required - or use an external redis server): 
+1. Get a Redis container (required - or use an external Redis server): 
```

Comment:
Not sure I like the way line 16 is written. My suggestion:
======================

Get a Redis container: 
`docker pull redis`

Note: Another option would be to use an external Redis server